### PR TITLE
Centers things and adds some padding

### PR DESCRIPTION
**Pull request by hbradio**
Usually the diff tool does a better job of showing only what really changed. But because my editor straightened up the tabs, pretty much everything is marked with red and green.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,28 +2,45 @@
 
 <title> Indesign, LLC </title>
 <link rel="stylesheet" type="text/css" href="style.css">
-<img src="https://scontent-iad3-1.xx.fbcdn.net/v/t1.0-9/30698735_1858784050841094_2508916175434416128_n.png?_nc_cat=0&oh=674b3def93de60f26fb30e8de65f5493&oe=5B64088C"
-    alt="Indesign, LLC">
```
Comment (hbradio) [inline index.html:6]: I moved the logo image down into the body.

File: style.css [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+#container {width: 1200px; margin-left: auto; margin-right: auto; padding: 0; background: #dddddd;}
+#logo-container {width: 1200px; margin-left: auto; margin-right: auto; margin-top: 20px; padding: 0; background: #ffffff;}
+#header {width: 1200px; height: 90px; margin: 0; padding-bottom: 20px; border: 0; background: #E0383D;}
+#sidebar {width: 200px; margin: 0; padding: 0; border: 0; float: left; background: #ffffff;}
+#content {width: 1000px; margin: 0; padding-bottom: 20px; border: 0; float: left; background: #ececec;}
+#footer {width: 1200px; height: 25px; margin: 0; padding: 0; border: 0; float: left; background: #414141; clear:both;}
+#content p {font-family: Arial; font-size: 14px; padding: 10px;}
+#sidebar {font-family: Arial; font-size: 18px}
+#footer {font-family: Arial; font-size: 18px; color: #dddddd}
+#group-photo img {display: block; margin: auto}
```
Comment (hbradio) [inline style.css:12]: Centering images is weird. I found a way to do it here: https://css-tricks.com/centering-css-complete-guide/

File: style.css [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-#sidebar {font-family: Arial; font-size: 18px}   
-#footer {font-family: Arial; font-size: 18px; color: #dddddd}  
-h1 {font-family: Arial; font-size: 24px; font-weight: normal; color:white}
+body {background: #ffffff; margin: 0; padding: 0;}
+a {color: rgb(255, 255, 255);}
+#container {width: 1200px; margin-left: auto; margin-right: auto; padding: 0; background: #dddddd;}
+#logo-container {width: 1200px; margin-left: auto; margin-right: auto; margin-top: 20px; padding: 0; background: #ffffff;}
+#header {width: 1200px; height: 90px; margin: 0; padding-bottom: 20px; border: 0; background: #E0383D;}
+#sidebar {width: 200px; margin: 0; padding: 0; border: 0; float: left; background: #ffffff;}
+#content {width: 1000px; margin: 0; padding-bottom: 20px; border: 0; float: left; background: #ececec;}
```
Comment (hbradio) [inline style.css:7]: I used `padding-bottom` to put some space between stuff.

**Comment by mhurlburt** [comment]
Thanks - that really made it look better!

**Pull request merged**